modules/scriptworker/templates/scriptworker.yaml.erb

```diff
@@ -17,6 +17,9 @@ verify_chain_of_trust: <%= @verify_chain_of_trust %>
 verify_cot_signature: <%= @verify_cot_signature %>
 cot_job_type: <%= @cot_job_type %>
 cot_product: <%= @cot_product %>
+# Calls to Github API are limited to 60 an hour. Using an API token allows to raise the limit to
+# 5000 per hour. https://developer.github.com/v3/#rate-limiting
+github_oauth_token: "<%= @github_oauth_token %>"
```

Comment:
I checked the empty works as I expect: scriptworker contacts the GitHub API without any token in the URL. This then limits the number of calls to 60 per hour. 

https://tools.taskcluster.net/groups/COzRCwrmTAiRVyFSvT0G0A/tasks/MbtI7uX-Sg6QV_FTxOQ47g/runs/2/logs/public%2Flogs%2Fchain_of_trust.log#L137